tests/test_objective_funs.py

```diff
@@ -93,6 +96,7 @@ def test(eq):
             W = obj.compute(
                 eq.R_lmn, eq.Z_lmn, eq.L_lmn, eq.p_l, eq.i_l, eq.c_l, eq.Psi
             )
```

Comment:
Looks like we can delete these lines since they are now redundant with line 99.

Reply:
yep, done